infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -129,8 +129,11 @@
                     + "                                                                          "
                     + " -k <seqcache>, --sequencer-cache-size=<seqcache>                         "
                     + "               The size of the sequencer's cache. [default: 250000].\n    "
-                    + " -B <size> --batch-size=<size>"
+                    + " -B <size> --batch-size=<size>                                            "
                     + "              The read/write batch size used for data transfer operations [default: 100].\n"
+                    + " -R <retention>, --metadata-retention=<retention>                         "
+                    + "              Max number of metadata files of each type to be retained    "
+                    + "on the disk [default: 100].\n"
```

Comment:
"Maximum number of system reconfigurations (i.e. layouts) retained for debugging purposes. "

Reply:
Done.